.orchestra/config/components/libedit.yml

```diff
@@ -1,32 +0,0 @@
-#@ load("@ytt:data", "data")
-#@ load("@ytt:overlay", "overlay")
-#@ load("/lib/create_component.lib.yml", "single_build_component")
-
-#@ source_url = "https://thrysoee.dk/editline/libedit-20191231-3.1.tar.gz"
-
-#@yaml/text-templated-strings
----
-#@ def libedit_args():
-license: COPYING
-configure: |
-  mkdir -p "$BUILD_DIR"
-  extract.sh --into "$BUILD_DIR" (@= source_url @)
-  cd "$BUILD_DIR"
-  ./configure \
-    --disable-examples \
-    --disable-static \
-    --enable-shared \
-    --prefix="$ORCHESTRA_ROOT"
-build_system: make
-build_dependencies:
-  - gcc-host-toolchain
-  - glibc
-dependencies:
-  - ncurses
-#@ end
-
-#@overlay/match by=overlay.all, expects=1
-#@overlay/match-child-defaults missing_ok=True
----
-components:
-  libedit: #@ single_build_component(**libedit_args())
```

Comment:
Please drop this commit, it's an optional dependency of both `llvm` and `clang-release`. I noticed it recently in an old llvm build directory which complained about missing libedit since we no longer pull it from the host system.

In fact, can you add it to those components?

Reply:
Added it back and enabled it in both `clang-release` and `llvm`